ui/server/graphql/resolvers/index.ts

```diff
@@ -259,6 +261,30 @@ const resolvers = {
         return null;
       }
     },
+    roundInvitationLink: async (parent, { roundId }, { user }) => {
+      const isAdmin =
```

Comment:
Can we replace this with the normal isCollOrGroupAdmin?

Reply:
A group admin who is not the member of a round in that group, should not be able to invite users.